test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +41,31 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() throws Exception {
+        // empty list
+        assertNoNull();
+
+        // at least one object is null
+        assertHasNull(null, 1, null);
+        assertHasNull("", null);
+        assertHasNull("abc", null);
+
+        // all objects null
+        assertHasNull(null, null, null);
+        assertHasNull(null, null);
+
+        // no null objects
+        assertNoNull(1, 2, 3);
+        assertNoNull("a", "b", "c");
```

Comment:
As discussed in the tutorial, these test cases can be separated into multiple more specific methods. Refer to our coding standard: https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html